support/src/testFixtures/java/se/inera/intyg/common/support/facade/testsetup/model/CommonMetaDataTest.java

```diff
@@ -52,6 +53,12 @@ void shouldIncludeType() {
         assertEquals(getInternalCertificate().getTyp(), metadata.getType());
     }
 
+    @Test
+    void shouldIncludeTypeName() {
+        final var metadata = getMetaData();
+        assertTrue(metadata.getTypeName().length() > 0);
```

Comment:
I think it would be great if we could also verify that the correct typeName was added in the test. Eg:

```
protected abstract String getTypeName();

@Test
void shouldIncludeTypeName() {
final var metadata = getMetaData();
assertEquals(getTypeName(), metadata.getTypeName());
```

Reply:
Fixed!